app/src/main/java/com/openlattice/chronicle/preferences/EnrollmentSettings.kt

```diff
@@ -113,7 +113,7 @@ fun getDevice(deviceId: String): AndroidDevice {
 }
 
 fun setCrashlyticsUser(studyId: UUID, participantId: String, deviceId: String) {
-    Crashlytics.setUserIdentifier(participantId)
-    Crashlytics.setUserEmail("$participantId@$studyId")
-    Crashlytics.setUserName(deviceId)
+    val crashlytics = FirebaseCrashlytics.getInstance()
+
+    crashlytics.setUserId(participantId)
```

Comment:
It's helpful with debugging to know the study id and device id. Isn't there a way to keep this information? Maybe this:

https://firebase.google.com/docs/crashlytics/customize-crash-reports?platform=android#add-keys

Reply:
I think setting custom keys might not be the right thing to do here. Custom keys help to get the **specific state of the app leading to a crash**. But since we are trying to identify users in crash reports, `setId` seems to be the only available option. 

Alternatively, we could create a custom id string that includes the deviceId, studyId and participantId. This seems to be the only viable option since `name` and `email` fields which were previously used by Fabric in addition to `Id` to identify users are not available in firebase crashlytics.